src/pages/home/_platform-slideshow.tsx

```diff
@@ -16,45 +14,9 @@ const ImagePlaceHolder = styled.div`
     }
 `
 
-const query = graphql`
-    query {
-        hero_platform1: file(relativePath: { eq: "home/hero_platform1.png" }) {
-            ...homePageHeroFadeIn
-        }
-        hero_platform1_eu: file(relativePath: { eq: "home/hero_platform1_eu.png" }) {
-            ...homePageHeroFadeIn
-        }
-        hero_platform2: file(relativePath: { eq: "home/hero_platform2.png" }) {
-            ...homePageHeroFadeIn
-        }
-        hero_platform2_eu: file(relativePath: { eq: "home/hero_platform2_eu.png" }) {
-            ...homePageHeroFadeIn
-        }
-        hero_platform3: file(relativePath: { eq: "home/hero_platform3.png" }) {
-            ...homePageHeroFadeIn
-        }
-        hero_platform4: file(relativePath: { eq: "home/hero_platform4.png" }) {
-            ...homePageHeroFadeIn
-        }
-    }
-`
-
-const StyledImage = styled(QueryImage)<{ $is_hidden: boolean }>`
+const ImageWrapper = styled.div<{ $is_hidden: boolean }>`
     opacity: ${({ $is_hidden }) => ($is_hidden ? '0' : '1')};
     display: ${({ $is_hidden }) => ($is_hidden ? 'none' : 'block')};
-    animation: fade 1s ease-in-out;
-
-    @media ${device.tabletL} {
```

Comment:
is this related to the improvement that you made?
i mean not disabling animation can be different from the thing that is in the scope of this task.

Reply:
the animation adds to UX feel of longer loading. I will add it back if any of the QA complains 😆

Reply:
correct